common/autoware_overlay_rviz_plugin/autoware_minimap_overlay_rviz_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+cmake_minimum_required(VERSION 3.8)
+project(autoware_minimap_overlay_rviz_plugin)
+
+set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};/usr/share/cmake/geographiclib")
+
+find_package(autoware_cmake REQUIRED)
```

Comment:
Thanks for the work. It looks great .
I get this error: ``` libautoware_minimap_overlay_rviz_plugin.so: undefined symbol: _ZN13GeographicLib6UTMUPS7ForwardEddRiRbRdS3_S3_S3_ib ```
and I added this line here --> find_package(GeographicLib REQUIRED)
it worked for me. Could you test and check .